dockerfiles/Dockerfile.training

```diff
@@ -11,9 +11,10 @@ ARG PYTORCH_VERSION=1.6.0
 
 ARG BUILD_VERSION=Release
 ARG OPENMPI_PATH=/opt/openmpi-${OPENMPI_VERSION} 
+ARG COMMIT_SHA1=master
```

Comment:
nit: "master" is not a SHA1 value. maybe refer to it as "commit" instead?

Reply:
Changed to "commit" now